kin-devplatform-sdk/src/main/java/kin/devplatform/data/blockchain/BlockchainSourceImpl.java

```diff
@@ -181,6 +187,9 @@ String generateMemo(@NonNull final String orderID) {
 
 
 	private void initBalance() {
+		if (balanceObserversCount > 0) {
+			startBalanceListener();
```

Comment:
why we didn't do it also in the past? why do we now start the listener?

Reply:
removed there's no need, the listener will be started on demand when the first listener will be added (addBalanceObserver)